haskell/gtk-gui/stack.yaml

```diff
@@ -0,0 +1,4 @@
+resolver: lts-21.13
```

Comment:
Is there a reason for using LTS 21.13 (GHC 9.4.7) instead of 22.13 (GHC 9.6.4)? Would it work with 22?

Reply:
Didn't notice this but we can bump to the 22.13 @jlgalarza3

Reply:
DONE! It works with 22.13